test/e2e/framework/framework.go

```diff
@@ -169,16 +172,64 @@ func (f *Framework) BeforeEach() {
 
 }
 
+func awaitNameSpace(f *Framework, cluster int) *v1.Namespace {
```

Comment:
You don't really need to return the Namespace as the caller doesn't use it.

Reply:
Done